### PR TITLE
ZS-189, removed MicrogridsResponseMessageFinder and updated Messaging…

### PR DESCRIPTION
**Pull request by robinbakkeruscgi**
…Config

### CODE REVIEW

File: osgp-adapter-ws-microgrids/src/main/java/com/alliander/osgp/adapter/ws/microgrids/application/config/MessagingConfig.java [hunk 1]
```diff
@@ -11,26 +11,24 @@
...
 import com.alliander.osgp.adapter.ws.infra.jms.LoggingMessageSender;
 import com.alliander.osgp.adapter.ws.microgrids.infra.jms.MicrogridsRequestMessageSender;
-import com.alliander.osgp.adapter.ws.microgrids.infra.jms.MicrogridsResponseMessageFinder;
 import com.alliander.osgp.adapter.ws.microgrids.infra.jms.MicrogridsResponseMessageListener;
 import com.alliander.osgp.shared.application.config.AbstractMessagingConfig;
 import com.alliander.osgp.shared.application.config.jms.JmsConfiguration;
 import com.alliander.osgp.shared.application.config.jms.JmsConfigurationFactory;
 
 @Configuration
-@PropertySources({ @PropertySource("classpath:osgp-adapter-ws-microgrids.properties"),
```
Comment (robinbakkeruscgi) [inline osgp-adapter-ws-microgrids/src/main/java/com/alliander/osgp/adapter/ws/microgrids/application/config/MessagingConfig.java]: this fiixed a sonar issue

File: osgp-adapter-ws-microgrids/src/main/java/com/alliander/osgp/adapter/ws/microgrids/application/config/MessagingConfig.java [hunk 2]
```diff
@@ -11,26 +11,24 @@
...
-        @PropertySource(value = "file:${osgp/Global/config}", ignoreResourceNotFound = true),
-        @PropertySource(value = "file:${osgp/AdapterWsMicrogrids/config}", ignoreResourceNotFound = true), })
+@PropertySource("classpath:osgp-adapter-ws-microgrids.properties")
+@PropertySource(value = "file:${osgp/Global/config}", ignoreResourceNotFound = true)
+@PropertySource(value = "file:${osgp/AdapterWsMicrogrids/config}", ignoreResourceNotFound = true)
 public class MessagingConfig extends AbstractMessagingConfig {
 
     @Autowired
-    public MicrogridsResponseMessageListener microgridsResponseMessageListener;
+    private MicrogridsResponseMessageListener microgridResponseMessageListener;
```
Comment (robinbakkeruscgi) [inline osgp-adapter-ws-microgrids/src/main/java/com/alliander/osgp/adapter/ws/microgrids/application/config/MessagingConfig.java]: renamed this property because Sonar complained anout name clash.

Comment (riannefijn) [inline osgp-adapter-ws-microgrids/src/main/java/com/alliander/osgp/adapter/ws/microgrids/application/config/MessagingConfig.java]: The 'private' can stay. Please return the 's'. This is something Sonar complains about only locally for you, and although the rule holds some truth, we should not all use different rules locally, but as a team/across teams all use the same rules. If you want to, you can take it up with the teams to discuss adding this rule to the set of rules we all use...

File: osgp-adapter-ws-microgrids/src/main/java/com/alliander/osgp/adapter/ws/microgrids/application/config/MessagingConfig.java (hunk 1 above)
Comment (riannefijn) [inline osgp-adapter-ws-microgrids/src/main/java/com/alliander/osgp/adapter/ws/microgrids/application/config/MessagingConfig.java]: If the sonar error is only locally, please revert the change. Although the rule holds some truth, we should not all use different rules locally, but as a team/across teams all use the same rules. If you want to, you can take it up with the teams to discuss adding this rule to the set of rules we all use...
